packages/ui/SurveysList/index.tsx

```diff
@@ -29,6 +29,12 @@ export default function SurveysList({
 }: SurveysListProps) {
   const [filteredSurveys, setFilteredSurveys] = useState<TSurvey[]>(surveys);
   const [orientation, setOrientation] = useState("grid");
+
+  useEffect(() => {
```

Comment:
Can we also call this at the end of the onboarding to remove them once instead of accessing the local storage on every survey list page call?